tests/test_main.py

```diff
@@ -1025,3 +1026,21 @@ class FunctionModel(BaseModel):
 
     m = FunctionModel()
     assert m.uid is uuid4
+
+
+@pytest.mark.skipif(sys.version_info < (3, 7), reason='field constraints are set but not enforced with 3.6 and above')
```

Comment:
surely this can be removed then?

Reply:
As a matter of fact it seems it can't :/